spiel/readme.txt

```diff
@@ -42,7 +42,11 @@ GNU General Public License for more details.
 
 This theme bundles the following third-party resources:
 
-All the images seen on the screenshot.png were generated with AI using Midjourney and DALL-E.
```

Comment:
These changes should be kept too, they're from another recent PR: https://github.com/Automattic/themes/pull/7554